app/helpers/validation_helper.rb

```diff
@@ -149,7 +181,7 @@ module ValidationHelper # rubocop:todo Metrics/ModuleLength
     preferred_contact_method: ['E-mailed Web Link', 'SMS Texted Weblink', 'Telephone call', 'SMS Text-message', 'Opt-out', 'Unknown', nil, ''],
     primary_telephone_type: ['Smartphone', 'Plain Cell', 'Landline', nil, ''],
     secondary_telephone_type: ['Smartphone', 'Plain Cell', 'Landline', nil, ''],
-    preferred_contact_time: ['Morning', 'Afternoon', 'Evening', nil, ''],
+    preferred_contact_time: TIME_OPTIONS.keys + [nil, ''],
```

Comment:
My bad for missing this in my suggestion, `TIME_OPTIONS.KEYS` doesn't quite work because the keys are symbols. They need to be stringified.

Reply:
no worries! done: https://github.com/SaraAlert/SaraAlert/pull/1019/commits/b18feb3b41c3bf2b6853604d1e1bff5b4c1302d2